<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stammbaum</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Stammbaum</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_mitglied">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>neuer Vorfahre</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>479</width>
       <height>243</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="label_5_begraben">
        <property name="text">
         <string>begraben</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7_kinder">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Anzahl Kinder</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3_ort">
        <property name="text">
         <string>Geburtsort</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QRadioButton" name="radioButton_maennlich">
        <property name="text">
         <string>männlich</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2_geb">
        <property name="text">
         <string>Geburtsdatum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEdit_2_geb"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_1_name">
        <property name="text">
         <string>Vorname, Nachname, (geborene(r))</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_7_kinder">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6_beruf">
        <property name="text">
         <string>Beruf</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_1_name"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_6_beruf"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3_ort"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8_geschlecht">
        <property name="text">
         <string>Geschlecht</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_5_begraben"/>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit_4_gestorben"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4_gestorben">
        <property name="text">
         <string>gestorben</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QRadioButton" name="radioButton_weiblich">
        <property name="text">
         <string>weiblich</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_neu">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>neuer Vorfahre</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_bearbeiten">
        <property name="text">
         <string>Vorfahre bearbeiten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_bearbeiten"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_vorfahren">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Eltern</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_vater">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>None (M)</string>
        </property>
        <item>
         <property name="text">
          <string>None (M)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_mutter">
        <item>
         <property name="text">
          <string>None (F)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_mutter">
        <property name="text">
         <string>Mutter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_vater">
        <property name="text">
         <string>Vater</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>239</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_accept">
        <property name="text">
         <string>Bestätigen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Abbrechen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1_name</tabstop>
  <tabstop>dateEdit_2_geb</tabstop>
  <tabstop>lineEdit_3_ort</tabstop>
  <tabstop>dateEdit_4_gestorben</tabstop>
  <tabstop>lineEdit_5_begraben</tabstop>
  <tabstop>lineEdit_6_beruf</tabstop>
  <tabstop>lineEdit_7_kinder</tabstop>
  <tabstop>comboBox_vater</tabstop>
  <tabstop>comboBox_mutter</tabstop>
  <tabstop>button_accept</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
